.github/workflows/ci.yml

```diff
@@ -50,11 +50,19 @@ jobs:
             compiler: gcc
 
         include:
+          # Add appropriate variables for gcov version required. This will intentionally break
+          # if you try to use a compiler that does not have gcov set
+          - compiler: gcc
+            gcov: gcov
```

Comment:
Should we choose a better name for this variable? I am afraid it is confused with the gcovr input passed to setup-cpp